[2023-09-22T21:42:06.603+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name manual__2023-09-22T21:42:03.977630+00:00 [queued]>
[2023-09-22T21:42:06.614+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name manual__2023-09-22T21:42:03.977630+00:00 [queued]>
[2023-09-22T21:42:06.615+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-09-22T21:42:06.630+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2023-09-22 21:42:03.977630+00:00
[2023-09-22T21:42:06.636+0000] {standard_task_runner.py:57} INFO - Started process 292 to run task
[2023-09-22T21:42:06.641+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'manual__2023-09-22T21:42:03.977630+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/test2.py', '--cfg-path', '/tmp/tmp1b02xfdf']
[2023-09-22T21:42:06.650+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask get_name
[2023-09-22T21:42:06.697+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T21:42:06.770+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name manual__2023-09-22T21:42:03.977630+00:00 [running]> on host 6d8bf768d09b
[2023-09-22T21:42:09.774+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2023-09-22T21:42:03.977630+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-22T21:42:03.977630+00:00'
[2023-09-22T21:42:09.776+0000] {python.py:183} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2023-09-22T21:42:09.819+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, execution_date=20230922T214203, start_date=20230922T214206, end_date=20230922T214209
[2023-09-22T21:42:09.862+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-09-22T21:42:09.882+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
